coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -699,6 +706,26 @@ def set_dataset_user_ids(self, dataset_id, user_ids):
         batch.commit()
         log.debug(f"Wrote {len(user_ids)} users to dataset {dataset_id}")
 
+    def update_dataset_user_ids(self, dataset_id, user_ids):
+        """
+        Updates user ids for the given dataset.
+
+        :param dataset_id: Id of a dataset.
+        :type dataset_id: str
+        :param user_ids: list of user ids.
+        :type user_ids: list
+        """
+        if len(user_ids) > 0:
```

Comment:
Can you log a warning if this isn't true?
e.g. 
```
if len(user_ids) == 0:
    log.warning(...)
    return
```

Reply:
updated, thanks.